From 21c9ba0cbd2b96d7b738c68195a3864ed9ecec42 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Sun, 17 Mar 2019 20:51:14 +0100
Subject: [PATCH] configure: do not test for CXX, it's not needed

There is no C++ source file, so no reason to require a C++ compiler.

Reported-by: Peter Korsgaard <jacmet@uclibc.org>
Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>
[Retrieved from:
https://git.buildroot.net/buildroot/tree/package/vde2/0001-no-cxx.patch]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index babf52b32..2aedfd984 100644
--- a/configure.ac
+++ b/configure.ac
@@ -11,7 +11,6 @@ AC_CONFIG_HEADER([include/config.h])
 AC_CONFIG_LIBOBJ_DIR(src/common)
 
 # Checks for programs.
-AC_PROG_CXX
 AC_PROG_CC
 AC_PROG_INSTALL
 AC_PROG_LN_S
-- 
2.35.0

